frontend/cypress/README.md

```diff
@@ -217,4 +218,14 @@ make -e CYPRESS_VIDEO=true cypress-run
 
 ### Tests are flaking 
 
-Try waiting for kiali to finish loading all the in-flight data before proceeding on to the next action or command. [Waiting for the loading spinner](https://github.com/kiali/kiali/blob/1766f20035e67a072dd68167869e0ce2009b9bc6/frontend/cypress/integration/common/overview.ts#L45-L46) to disappear is a good enough measure for this.
\ No newline at end of file
+Try waiting for kiali to finish loading all the in-flight data before proceeding on to the next action or command. [Waiting for the loading spinner](https://github.com/kiali/kiali/blob/1766f20035e67a072dd68167869e0ce2009b9bc6/frontend/cypress/integration/common/overview.ts#L45-L46) to disappear is a good enough measure for this.
+
+To run only specific subset of the test suite, tag the Gherkin scenarios with the `@selected` tag. You can then run these using:
```

Comment:
can you provide a sample here of what it would look like using the selected tag? Just a simple block of code that tags a test with selected to show people what it should look like.